Tests/private_build/upload_packs_private.py

```diff
@@ -66,57 +66,112 @@ def update_private_index(private_index_path: str, unified_index_path: str):
         shutil.copy(path_to_pack_on_unified_index, path_to_pack_on_private_index)
 
 
-def get_private_packs(private_index_path: str, pack_names: set = set(),
-                      extract_destination_path: str = '') -> list:
+def add_private_pack(private_pack_metadata, private_packs, changed_pack_id):
```

Comment:
personal convention:
the first argument of `add_private_pack` should be the `private_packs` as it acts as `self` - this is the object that the function modifies